SONAR-netCDF4/docs/crr341.adoc

```diff
@@ -415,9 +415,11 @@ Type 4 conversion equations are intended for data produced by Simrad EK80 and ES
 
 == Coordinate systems
 
-This section provides a complete mathematical overview of the coordinate systems necessary to physically locate the position of an echo relative to a fixed global coordinate system when measured from a moving or stationary platform. The coordinate systems detailed here follow those used in some multibeam bathymetric mapping sonars, but in many cases this level of preciseness will be unnecessary and several coordinate systems will overlap.
+This section provides a complete mathematical overview of the coordinate systems necessary to physically locate the position of an echo relative to a geographic coordinate system when measured from a moving or stationary platform. The coordinate systems detailed here follow those used in some multibeam bathymetric mapping sonars, but in many cases this level of preciseness will be unnecessary and several coordinate systems will overlap.
 
-There are five right-handed Cartesian coordinate systems (<<coordinateSystemList>>) and one Cartesian angular coordinate system used to specify the physical location of received echoes. Some of the coordinate systems have different origins and this is implemented as a translation vector in the input coordinate system. Rotation angles are as per the right-hand rule.
+There are four right-handed Cartesian coordinate systems (<<coordinateSystemList>>, <<coordinate_system_figure>>) associated with the platform, transducer(s), and acoustic beams, and one Cartesian angular coordinate system used to specify the physical location of received echoes with a sonar beam. Some of the coordinate systems have different origins and this is implemented as a translation vector in the input coordinate system. Rotation angles are as per the right-hand rule.
+
+There is also a geographic coordinate system that provides for location of the platform on Earth and also the height above a datum (e.g. mean sea level). The platform heading variable (degrees clockwise from north) can be used to obtain the sonar orientation in the geographic coordinate system (this applies to both stationary and mobile sonar platforms).
 
```

Comment:
There is also a geographic coordinate system that provides for location of the platform on Earth and also the height above a datum (e.g. mean sea level).
comment: see below, if we consider SCS coordinate system as a local coordinate system associated with the platform and with horizontal axis along North and East, the sonar orientation is obtained directly with the dynamic yaw of the attitude sensor which contains the heading

Reply:
ok